src/components/HomePage.js

```diff
@@ -69,7 +74,34 @@ type Props = {
   navigateToOpenID: (authFormData: Object, provider: Object) => void,
 };
```

Comment:
You need to add the type for `match` here also or else flow will complain (as you can see in the travis build)
```javascript
...
match: HomePageRouteMatch,
...
```